PsfLauncher/UniqueHkey.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <windef.h>
+#include <handleapi.h>
+#include <winreg.h>
+
+class UniqueHKEY

```

Comment:
>UniqueHKEY [](start = 6, length = 10)

fyi - wil is open source now:
[https://github.com/microsoft/wil/](https://github.com/microsoft/wil/)

Reply:
Added wil::uniquehkey